demos/text-field.html

```diff
@@ -139,7 +139,7 @@ <h2>Password field with validation</h2>
         <h2>Outlined Text Field</h2>
         <div id="demo-tf-outlined-wrapper">
           <div id="tf-outlined-example" class="mdc-text-field mdc-text-field--outlined" data-demo-no-auto-js>
-            <input required pattern=".{8,}" type="text" id="tf-outlined-input" class="mdc-text-field__input"
+            <input pattern=".{8,}" type="text" id="tf-outlined-input" class="mdc-text-field__input"
```

Comment:
This line is inconsistent with the JS code below (pattern would be `.{8,}` when required, but `.*` when not)

Reply:
yes you're correct. I'll sift through the rest of the file and make sure that all works the same.